aws/ecs/cloudformation-identiorca.json

```diff
@@ -155,16 +157,12 @@
         "Yes"
       ]
     },
-    "UseScopeService": {
-      "Fn::Not": [
+    "UseWeaveScopeStandalone": {
```

Comment:
Wouldn't it be simpler to re-use the WeaveCloudServiceToken and check if it's empty in the ELB listener section?